source/tutorials/filtering.rst

```diff
@@ -129,6 +138,15 @@ For example, filtering the table to contain only samples from subject 1 is perfo
       --p-where "Subject='subject-1' AND BodySite='gut'" \
       --o-filtered-table subject-1-gut-filtered-table.qza
 
+The ``OR`` keyword syntax is similar to the ``AND`` keyword syntax, but specifies that either of the expressions can be true for a sample to be retained. For lack of a more relevant application to the example data being used here, the ``OR`` keyword in this example is applied to retain samples all of the samples where ``BodySite`` is ``gut`` *or* ``ReportedAntibioticUsage`` is ``Yes`` in ``sample-metadata.tsv``.
```

Comment:
This seems like a good example to me (agree it is a bid contrived, but it makes sense). My only thought is to explore some other variation, like `DaysSinceExperimentStart='140' OR ReportedAntibioticUsage='Yes'`, that way we are pulling in a metadata category that has generally been untouched in the tutorial.